src/io/parser.rs

```diff
@@ -75,6 +75,34 @@ pub fn multispace_or_comment1(input: &str) -> IntermediateResult<()> {
     value((), many1(alt((value((), multispace1), comment))))(input)
 }
 
+fn parse_bare_name(input: &str) -> IntermediateResult<&str> {
+    traced(
+        "parse_bare_name",
+        recognize(pair(
+            alpha1,
+            opt(pair(
+                many0(tag(" ")),
+                separated_list1(
+                    many1(tag(" ")),
+                    many1(satisfy(|c| {
+                        ['0'..='9', 'a'..='z', 'A'..='Z', '-'..='-', '_'..='_']
+                            .iter()
+                            .any(|range| range.contains(&c))
+                    })),
```

Comment:
Shouldn't this be the same?
```rust
                many0(satisfy(|c| {
                    ['0'..='9', 'a'..='z', 'A'..='Z', '-'..='-', '_'..='_', ' '..=' ']
                        .iter()
                        .any(|range| range.contains(&c))
                })),
```

Reply:
I think then bare names could have trailing spaces.